### DISCUSSION TITLE
High Ping on my home server?

### DISCUSSION THREAD

**Discussion by AlvaroMartinezB** [jagrosh/MusicBot#815] [Troubleshooting]
I know this probably isn't an issue with the bot's code per se, but I'd like to know any tips or advice you might have. 

My bot is being hosted on my home server, it has an ethernet connection to my 300/100 fiber. The same network that my gaming PC is connected to.  The bot seems to have a high ping, all the time, sometimes a high Websocket ping. I would not mind the high ping if it didn't cause noticeable quality loss. You can clearly hear the music, but it skips now and then, and seems to have some packet loss. Here are some images for reference, as well as the ping I have on my discord app on my computer (keep in mind they are on the same network). I will appreciate any feedback you might have!

![image](https://user-images.githubusercontent.com/14044330/133564538-705e0a16-bc6b-4b6c-adb2-efd28459edaf.png)

![image](https://user-images.githubusercontent.com/14044330/133564624-6ee5831e-9d15-4648-b362-1baa17dbaee5.png)

![image](https://user-images.githubusercontent.com/14044330/133564572-0b855908-30a1-47b8-ae48-593a145618b1.png)

**Comment by YJSoft** [comment]
https://github.com/JDA-Applications/JDA-Utilities/blob/master/examples/src/main/java/com/jagrosh/jdautilities/examples/command/PingCommand.java#L44-L49

It's not voice channel ping. It just checks time diff.

**Comment by MichailiK** [comment]
As YJSoft said, the ping command only checks the ping for text messages, not voice.

Could you run the debug command so we may have an idea of your environment?
